src/seaport.ts

```diff
@@ -1149,6 +1436,48 @@ export class OpenSeaPort {
     return this.validateAndPostOrder(orderWithSignature);
   }
 
+  /**
+   * Fullfill or "take" an order for an asset, either a buy or sell order
+   * NOTE: "Gifting" (fulfilling with recipient address) is not yet supported
+   * NOTE: Fulfilling private listings is not yet supported
+   * @param options fullfillment options
+   * @param options.order The order to fulfill, a.k.a. "take"
+   * @param options.accountAddress The taker's wallet address
+   * @param options.recipientAddress The optional address to receive the order's item(s) or curriencies. If not specified, defaults to accountAddress.
+   * @returns Transaction hash for fulfilling the order
+   */
+  public async fulfillOrder({
+    order,
+    accountAddress,
+  }: {
+    order: OrderV2;
+    accountAddress: string;
+    // TODO: Implement recipientAddress
+    recipientAddress?: string;
```

Comment:
If it's not supported, maybe we should comment out the input param for now? Otherwise it could be used by accident if they don't read the comments.